docs/ABOUT.md

```diff
@@ -9,7 +9,7 @@ This is often invisible to the beginning programmer, but if there are uses for w
 
 Python was started by Guido van Rossum in 1989.
 Its name is an homage to the comedy troupe Monty Python. 
-Python 2 is used widely but support may end by 2020.
+Python 2 is used widely but support [may end by 2020](https://pythonclock.org/).
```

Comment:
I like the idea of adding a link to validate this statement, but would be more in favor of linking to [PEP-373](https://www.python.org/dev/peps/pep-0373/#id2) since it comes from an official source (pythonclock.org is not owned or maintained by the Python Foundation).

@N-Parsons thoughts?

Reply:
@cmccandless Sorry for the late response. I can update this with a new commit soon, if the PEP-373 link is preferred!

Reply:
Yeah, an official source seems like a better choice here. Thanks!

Reply:
@jflory7 were you still going to change the link to an official source?